src/main.ts

```diff
@@ -7,9 +7,20 @@ import { printer } from './print/index';
 
 const languages: SupportLanguage[] = [
   {
-    extensions: ['.gjs', '.gts'],
-    name: 'Ember Template Tag',
+    name: 'Ember Template Tag (gjs)',
+    aliases: ['gjs', 'glimmer-js'],
```

Comment:
since `glimmer-js` and `glimmer-ts` are the VSCode names for these languages, do ya'll have any reservations about also adding `glimmer.javascript` and `glimmer.typescript`? (this is what neovim / tree-sitter use) -- though... I'm not entirely sure it matters since to hook up prettier in neovim anyway, you have some non-default things you gotta do. :thinking: 

In anycase, I'm curious how specific `vscodeLanguageIds` is specific to `vscode`, or if it's something that all language-server-interpreting tools respect, or if other editors can/should be added to here as well :thinking:

Reply:
Would any of these help? They’re all undocumented besides the list here 🙃

https://prettier.io/docs/en/api#prettiergetsupportinfo

Reply:
I'm not sure tbh. I've yet to find a neovim plugin that integrates with prettier _and_ automatically configures which files to run it on :sweat_smile: